integrations/access/opsgenie/testlib/suite.go

```diff
@@ -184,6 +186,34 @@ func (s *OpsgenieSuiteOSS) TestAlertCreationForTeams() {
 	assert.Equal(t, alert.ID, pluginData.AlertID)
 }
 
+func (s *OpsgenieSuiteOSS) TestPriorityOverride() {
+	t := s.T()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	t.Cleanup(cancel)
```

Comment:
Is this necessary, or can we just use `context.Background()`? Go tests will time out automatically and I don't see anything here that is likely to cause a timeout.